ui/pnpm-lock.yaml

```diff
@@ -0,0 +1,6048 @@
+lockfileVersion: '6.0'
```

Comment:
> oh damn I used `pnpm` instead `npm` heh...

I'm not really up on the JS ecosystem. Is `pnpm` better enough to be worth switching?
* If so, we should update everything to match: instructions in `guide/build.md`, GitHub workflows, removing the `package-lock.json`
* otherwise, we should kill this file. It'll be confusing to have both, especially without anything guaranteeing they match.

Reply:
`release.yml` is still using `npm`, so it's just not locked now and the product might be something different than what was seen in any pre-release testing. We should switch it, too. I think I'd also just have `guide/install.md` say to use `pnpm` too because if you do the `npm` path, you likewise end up with something a little different than what anyone else is running with.

Reply:
Hmmm, so we will mainly use `pnpm` right?